### ISSUE TITLE
Sound issues

### ISSUE THREAD

**Issue by melissay94** [RITAccess/data-analyzer#148]
There are instruments that do not work so well with the data that need to be removed. Some examples are the Helicopter and Fx3 (crystal). There is also a very small range of notes that will play and we should look into either expanding it or finding a better algorithm for the sounds. There were also some issues with slowing down JAWS.

**Comment by smcintosh881** [comment]
Should we play through each instrument and delete them?

**Comment by jmiller656** [comment]
I don't think we necesarily need to delete them. There already is math being done inside my instrument code in order to fit the notes to a certain range. The problem is that not all midi instruments use all of the 127 notes. I have a link here that contains note ranges for most midi instruments: http://soundprogramming.net/file-formats/midi-note-ranges-of-orchestral-instruments/

**Comment by Rek3650** [comment]
Im still confused about what needs to be done here. Do you want us to remove the less useful sounds like the Helicopter while writing a new algorithm that takes into account the note ranges of different instruments?

**Comment by jmiller656** [comment]
Ok, so as of now, I have implemented a solution for instruments which restricts many of the instruments to their specific range (those that I could find, listed in the link in my previous comment). The rest of the instruments have a default range from 30-100 which seems to fit most of the other oddball instruments that weren't included in the aforementioned list. At some point, we should probably conduct the tedious task of testing all of the instruments to make sure that the default range works for the ones we don't have a specific range for. I will warn ahead of time that if there are wide variations in the data (ex a set containing [1,1,1,1,1000,100000, 999999999999]), the user will probably get only two different notes (maybe three for this specific case), those being the maximum and minimum note in the instrument's range. Basically what my code does is it takes the set, and if it isn't already in the current instrument's range, it will adjust everything in the instrument's copy of the data in order to fit into the instrument's range with the same relative distance between pitches. This is in the buildnotes function of Instrument
